### PR TITLE
[Connection] Fix connection icon load from connection definition

### PR DESCRIPTION
**Pull request by senali-d**
**Notes for Reviewers**

This PR fixes #

- [x] Typo issue "transition"
- [x] Meshery model name change to "meshery-core"
- [x] Fix connection icon loading issue

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

**Review by MUzairS15 — APPROVED** [review]

File: ui/components/connections/index.js [hunk 1]
```diff
@@ -248,9 +248,7 @@ function Connections({
                   ping(tableMeta.rowData[3], tableMeta.rowData[2], tableMeta.rowData[0]);
                 }
               }}
-              iconSrc={
-                getColumnValue(tableMeta.rowData, 'kindLogo', columns)?.colorIcon.split('public')[1]
-              }
+              iconSrc={getColumnValue(tableMeta.rowData, 'kindLogo', columns)?.colorIcon}
```
Comment (MUzairS15) [inline ui/components/connections/index.js:251]: colorIcon??

Comment (senali-d) [reply ui/components/connections/index.js]: @MUzairS15 Added icon according to the color mode.

File: ui/pages/_app.js [hunk 2]
```diff
@@ -228,8 +228,8 @@ class MesheryApp extends App {
         connectionDef[CONNECTION_KINDS[kind]] = {
           transitions: res?.components[0].model.metadata.transitions,
           icon: {
-            colorIcon: res?.components[0].model.metadata.svgColor,
-            whiteIcon: res?.components[0].model.metadata.svgWhite,
+            colorIcon: res?.components[0].model.metadata.svgColor.split('public')[1],
```
[the remark this answers is not in the document]
Comment (senali-d) [reply ui/pages/_app.js:235]: Without split logic icon was not loaded. I have already tested it after build. It loads properly once remove the `ui/public`

**Review by leecalcote — APPROVED** [review]

File: ui/pages/_app.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (leecalcote) [reply ui/pages/_app.js:235]: @MUzairS15 as this PR is merged, is a follow up issue needed for reconfiguration of dev env?

**Pull request merged**